### ISSUE TITLE
Refactoring of AccountSelector dropdown #15099 - Prepare and execute tests

### ISSUE THREAD

**Issue by virginiabalducci** [status-im/status-desktop#15232]
https://github.com/status-im/status-desktop/pull/15099

https://www.notion.so/AccountSelector-dropdown-refactor-testing-15099-f72e4d0eedce4f518d6ca10b50e51ef0

**Comment by alexjba** [comment]
@virginiabalducci Found another issue where the preselected account in the wallet view was not automatically selected when opening the send, receive and swap modal. It's fixed in the latest builds

**Comment by alexjba** [comment]
Automatically closed by accident.
Please update https://github.com/status-im/status-desktop/issues/14691 once testing is done.
